src/rgw/rgw_sal_motr.cc

```diff
@@ -1482,11 +1468,25 @@ int MotrObject::MotrReadOp::iterate(const DoutPrefixProvider* dpp, int64_t off,
 {
   int rc;
 
-  if (source->category == RGWObjCategory::MultiMeta)
+  if (source->category == RGWObjCategory::MultiMeta){
```

Comment:
minor one, there should be space b/w `)` and `{`

`if (source->category == RGWObjCategory::MultiMeta) {`

Reply:
done. please check